Docker/CI.Ubuntu22.dockerfile

```diff
@@ -0,0 +1,41 @@
+FROM ubuntu:22.04
```

Comment:
(nit)
I have an impression that 
`Docker/CI.Ubuntu20.dockerfile` has been renamed to
`Docker/CI.Ubuntu22.dockerfile` and some changes were made in the file content. 
But the result of those changes is shown such that the old file was completely removed and a brand new file has been created (the commit history is not preserved through the renaming). This typically happens when renaming is done with commands bypassing the git (rather than command `git mv old_path/old_name new_path/new_name`).

Just in case I'd like to make sure that you also considered the following alternatives.
A. If you want to preserve the commit history you can use git for renaming, 
`git mv Docker/CI.Ubuntu20.dockerfile Docker/CI.Ubuntu22.dockerfile`
then commit the change, then make changes to the file content, then commit the content change.

B. Renaming to 
`Docker/CI.Ubuntu.dockerfile`, such that the version of Ubunto is not in the file name, and you don't need to rename the file any more upon migration to newer Ubunto. You just make changes to the file itself.

C. Just adding 
`Docker/CI.Ubuntu22.dockerfile`, but not removing 
`Docker/CI.Ubuntu20.dockerfile`, such that the support for Ubuntu22 is added, but the support for Ubuntu20 is not removed.

Reply:
As such you are right that it is a rename and the appropriate way to have done that would be to use `git mv`. In this case, we are removing support for Ubuntu 20 as there is no straight forward path to install LLVM 13 on Ubuntu 20. I will keep `git mv` in mind for next time.